block/src/Pos/Block/Network/Logic.hs

```diff
@@ -107,30 +102,6 @@ triggerRecovery diffusion = unlessM recoveryInProgress $ do
            throwM e
     logDebug "Finished requesting tips for recovery"
 
-requestTipOuts :: BlockInstancesConstraint m => Proxy m -> OutSpecs
-requestTipOuts _ =
-    toOutSpecs [ convH (Proxy :: Proxy MsgGetHeaders)
```

Comment:
This seems to have been removed altogether. It doesn't seem to have been moved elsewhere in this commit.

Is that intentional?

Reply:
Yes. It wasn't used, but since it was exported, GHC couldn't warn of an unused definition.

Reply:
It's part of a long-standing annoying design wart: the application-level "worker" things used to define the network request types that they were capable of making, and the whole network interface of the program was inferred from this set of workers. Now the network interface is encapsulated in a `Diffusion`, and it really make no sense to speak of these `OutSpecs` or `InSpecs` anywhere else. I intend to purge them soon but it will probably lead to another >1000 line change :)